components/decisioncard.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Head from "next/head";
+import Textbox from "../components/textbox";
+import styles from "../styles/decisioncard.module.scss";
+
+export default function DecisionCard() {
+  return (
+    <>
+      <Head>
+        <title>Decision Card Container</title>
+      </Head>
+      <div id="decisioncardcontainer" className={styles.CardAlpha}>
```

Comment:
this is a v picky comment, but just for awareness the norm for class names is kebab-case for readibility (so decision-card-container)